data/sheragi/archaeology missions.txt

```diff
@@ -212,19 +212,37 @@ mission "Deep Archaeology 5"
 		conversation
 			`You breathe a sigh of relief as you land safely in the spaceport on <planet>. "Okay, what was that all about?" you ask.`
 			`	"I honestly don't know," says Albert. "I can tell you what I suspect, though. The history books don't actually say when the Deep was first settled. The first documented expedition to the Deep was in the 24th century, and they found several human colonies already established in the region, claiming to date back to the very dawn of space flight. But I think the first settlers were brought there long before that. Brought there on alien ships."`
+			branch "pug"
+				or
+					has "FWC Pug 2: offered"
+					has "FW Pug 3: offered"
 			choice
 				`	"For what purpose?"`
 					goto purpose
 				`	"What aliens? The Quarg?"`
 					goto aliens
+			label pug
+			choice
+				`	"For what purpose?"`
+					goto purpose
+				`	"Wait, are you talking about the Pug?"`
+					goto surprise
 			label purpose
 			`	"I'm not sure," he says. "But I first became interested in the Deep while researching the Alpha Wars. I'm sure you know the history, how humanity was nearly enslaved by psychopathic 'super' humans that we ourselves had created by dabbling in genetic engineering. At that time, the Deep made a sudden, improbable technological leap that allowed them to construct weapons and ships far more powerful than what the Alphas had, and the war against them was won."`
 				goto end
 			label aliens
 			`	"There are popular folk tales in the Deep about 'elves,'" says Albert. "Thin, pale, graceful creatures, with blue blood."`
 			`	John frowns. "Hemocyanic blood. Not the Quarg, then. Their lungs are odd, but their blood uses hemoglobin for oxygen transport, just like ours."`
+				goto end
+			label surprise
+			`	Albert and John look at each other confusedly before Albert says, "Who are the Pug, exactly?"`
```

Comment:
The Pug invading human space is a fairly public event, right? Why wouldn't Albert and John already know about it? I can think of two ways you could change the branch in light of Albert and John probably already knowing about the Pug:
1. On one hand, you could just have them not know much about the Pug's description, and that wouldn't be much rewriting. However, I do find it hard to believe that they wouldn't have already known that too if they knew about the Pug.
2. In that case, maybe the best thing to do would be to cut the part where `<first>` explains who the Pug are and rewrite the later section to be Albert just explaining the connection between the Pug and folk tale aliens' descriptions.

I think I'd prefer the latter of these two options, but it's up to you.

Reply:
Of these options, I've elected to go with the latter. It's in the most recent commit.